e2e/main_test.go

```diff
@@ -107,33 +105,35 @@ func mockEnvoyVersionsServer() (*httptest.Server, error) {
 	return ts, nil
 }
 
-// readFuncEPath reads E2E_FUNC_E_BINARY or defaults to "$PWD/dist/func-e_$GOOS_$GOARCH/func-e"
+// readFuncEBin reads E2E_FUNC_E_PATH or defaults to "../func-e"
 // An error is returned if the value isn't an executable file.
-func readFuncEPath() (string, error) {
-	path := os.Getenv(funcEBinaryEnvKey)
-	if path == "" {
-		// Assemble the default created by "make bin"
-		relativePath := filepath.Join("..", "dist", fmt.Sprintf("func-e_%s_%s", runtime.GOOS, runtime.GOARCH), "func-e")
-		abs, err := filepath.Abs(relativePath)
-		if err != nil {
-			return "", fmt.Errorf("%s didn't resolve to a valid path. Correct environment variable %s", path, funcEBinaryEnvKey)
+func readFuncEBin() error {
+	path := os.Getenv(funcEPathEnvKey)
+	path = filepath.Join("..", path) // relative to project root
+	if path != "" {
```

Comment:
therefore it seems that we never reach this branch to me

Reply:
oh yeah lemme check that!

Reply:
this is ok. `..` will always exist.

Reply:
but you are right, the if isn't needed

Reply:
ah - I confused != with == 😄